packages/frontend/cypress/integration/3.sign_in_page_spec.js

```diff
@@ -0,0 +1,127 @@
+describe('Sign in page', () => {
+  const rand = Math.floor(Math.random() * (1000 - 100) + 100);
+  const email = `test${rand}@gmail.com`;
+
+  // clear all local storage before tests
+  // Should really perform resetting and seeding of database instead
+  beforeEach(() => {
+    indexedDB.deleteDatabase('firebaseLocalStorageDb');
+  });
+
+  // should route to /auth
+  it('should visits /auth', () => {
+    cy.visit('/auth');
+    cy.contains('Click here to sign up').click();
```

Comment:
Use of `cy.contains` is considered very bad practice, as your tests are decoupled from your code and instead depend on whatever text you have on the page (see https://docs.cypress.io/guides/references/best-practices). Place `data-cy` tags on the DOM elements you want to click, as this means your tests won't break if the text descriptors you've been using change.